lib/thrift/servers/binary_framed/acceptor_supervisor.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Thrift.Servers.BinaryFramed.AcceptorSupervisor do
+  @moduledoc """
+  A supervisor for acceptor processes.
+  """
+
+  alias Thrift.Servers.BinaryFramed.Acceptor
+  use Supervisor
+
+  def start_link(socket, server_module, handler_module, count) do
+    Supervisor.start_link(__MODULE__, [socket, server_module, handler_module, count])
+  end
+
+  def init([socket, server_module, handler_module, count]) do
+    children = for id <- (1..count) do
+      worker_id = "#{inspect handler_module}_acceptor_#{id}"
```

Comment:
`id` does not need to be an atom, I think `{handler, id}` would be a more natural data structure.